webapp/src/ts/modules/analytics/analytics.component.ts

```diff
@@ -44,30 +44,38 @@ export class AnalyticsComponent implements OnInit, OnDestroy {
   }
 
   private subscribeToStore() {
-    const selectorsSubscription = combineLatest(
-      this.store.select(Selectors.getAnalyticsModules),
-    )
-      .subscribe(([
-        analyticsModules = [],
-      ]) => {
+    const selectorsSubscription = combineLatest(this.store.select(Selectors.getAnalyticsModules))
+      .subscribe(([ analyticsModules = [] ]) => {
```

Comment:
```suggestion
    const selectorsSubscription = this.store.select(Selectors.getAnalyticsModules)
      .subscribe((analyticsModules = []) => {
```
I'm still getting used to RxJS but could we get rid of `combineLatest` here?

Reply:
Oh right! Since we only have 1 subscription. Done!